### DISCUSSION TITLE
Disable renaming of function/Component names in React

### DISCUSSION THREAD

**Discussion by SrinivasPrabhu794** [parcel-bundler/parcel#8891] [Q&A]
I have an existing PHP app, where I want to include a react-based application (as a library target) using parcel bundler.

Everything works great including generating of the source maps. But one thing is when I look at my react developer tools on chrome developer tools the component name seems to be renamed like this $dda87bff01a933ff$var$MyComponent instead of just being MyComponent

![image](https://user-images.githubusercontent.com/817112/225573255-21b7b6d5-e16e-44ff-a24b-a34699555a09.png)

Is there a way to disable this? My build script looks like this:

`"build": "parcel build --no-optimize --no-scope-hoist"`

Any help regarding this will be very valuable.

Thank you!

**Comment by devongovett** [comment]
#6092

**Reply by SrinivasPrabhu794** [reply]
So, this will be solved in a future release ?

**Reply by mischnic** [reply]
Also related: https://github.com/parcel-bundler/parcel/issues/8724
